libs/shared/ui/src/lib/components/error-boundary/error-boundary.tsx

```diff
@@ -0,0 +1,42 @@
+import { type PropsWithChildren } from 'react'
+import { ErrorBoundary as ErrorBoundaryRaw } from 'react-error-boundary'
+import { type FallbackProps } from 'react-error-boundary'
+import { useLocation } from 'react-router-dom'
+import { Button } from '../button/button'
+import { Heading } from '../heading/heading'
+import { Icon } from '../icon/icon'
+import { Section } from '../section/section'
+
+function ErrorFallback({ error, resetErrorBoundary }: FallbackProps) {
+  return (
+    <Section className="flex-grow items-center justify-center rounded-sm bg-white">
```

Comment:
I think we need to remove the `justify-center` and have an static margin top to avoid this kind of view

<img width="1708" alt="Screenshot 2024-06-05 at 16 25 14" src="https://github.com/Qovery/console/assets/533928/fde73cd4-8adb-4f0b-bef4-684c59b4ecc8">